Source/Extensions/Blazorise.Components/Autocomplete.razor.cs

```diff
@@ -578,30 +604,35 @@ where text.StartsWith( CurrentSearch, StringComparison.OrdinalIgnoreCase )
         /// <summary>
         /// Clears the selected value and the search field.
         /// </summary>
-        public async Task Clear()
+        public async Task ClearSelected()
```

Comment:
Since you've started using the `Reset` prefix, might as well rename it to `Reset()`. Since that's what it actually does right?